manifests/bundle.pp

```diff
@@ -13,6 +15,9 @@
     Class['katello_devel::rvm'] -> Exec["bundle-${title}"]
     $command = "rvm ${rvm_ruby} do bundle ${title}"
     $path = "/home/${user}/.rvm/bin:/usr/bin:/bin"
+  } elsif $use_scl_ruby {
+    $command = "scl enable ${scl_ruby} 'bundle ${title} --path .vendor'"
```

Comment:
Drop the `--path .vendor` here as this breaks `bundle exec`. Instead move it to here -- https://github.com/theforeman/puppet-katello_devel/blob/master/manifests/setup.pp#L25

I got a green install making that change.